old_syntax.js

```diff
@@ -0,0 +1,12 @@
+
+    <!-- old links -->
```

Comment:
Expected an identifier and instead saw '<'.<br>Expected an operator and instead saw '!'.<br>Expected an operator and instead saw '--'.<br>Expected an assignment or function call and instead saw an expression.